### ISSUE TITLE
Recommended development process for building docker images?

### ISSUE THREAD

**Issue by emdupre** [poldracklab/fmriprep#1796]
In investigating #1786, @shotgunosine and I tried to build new Docker images of fMRIprep as recommended in the [development guidelines](https://fmriprep.readthedocs.io/en/stable/contributors.html#rebuilding-docker-image). Unfortunately, running the built images fails with the error:

```python-traceback
Traceback (most recent call last):
  File "/usr/local/miniconda/bin/fmriprep", line 10, in <module>
    sys.exit(main())
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/cli/run.py", line 311, in main
    opts = get_parser().parse_args()
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/cli/run.py", line 49, in get_parser
    currentv = Version(__version__)
  File "/usr/local/miniconda/lib/python3.7/site-packages/packaging/version.py", line 221, in __init__
    raise InvalidVersion("Invalid version: '{0}'".format(version))
packaging.version.InvalidVersion: Invalid version: ''
```

Is there a new, recommended method for building images ? Here, we simply modified `setup.cfg` and created the image with

`docker build -t fmriprep-t2s .`

**Comment by effigies** [comment]
Ah, yeah, those do not reflect the current process:

https://github.com/poldracklab/fmriprep/blob/a6029eda7a418cea2d9f7326e6d455afa2c5acb4/.circleci/config.yml#L61-L67

And the critical one is `--build-arg VERSION`, and `THISVERSION` is just the output of `python get_version.py`. So:

```Shell
docker build -t fmriprep-t2s --build-arg VERSION=$( python get_version.py ) .
```

I'll update the docs. Thanks for pointing this out.

**Comment by effigies** [comment]
@emdupre If you care to review the updated docs: #1797.

**Comment by emdupre** [comment]
Closed with #1797
